### ISSUE TITLE
Can you even Emulate with UTM SE?

### ISSUE THREAD

**Issue by Acceleratepowershellhell** [utmapp/UTM#6687]
**Describe the issue**
Trying to Emulate Debian 12 on Iphone Xs but OS Freezes during Loading. I tried to load anything from Windows 11 to 7 and many other OS none do work when Emulating. As stated on the website, emulation is slower, but it doesn't say impossible to run. The IOS documentation on the website is a very lacking. Default Disk-Space is also below 3 Gb on this Debian 12 Configuration. 

**Configuration**
* UTM Version: 4.5.4
* OS Version:  17.6.1
* Device Model: Iphone Xs
* Jailbroken: No (Emulation)
* How did you istall UTM: Appstore

**Debug log**
[debug.log](https://github.com/user-attachments/files/17077656/debug.log)

**Used VM**
(https://mac.getutm.app/gallery/debian-12)

Tried:
Windows 11 (Converted)
Tiny 11 (Converted)
Windows 10 (Converted)
Windows 7 (UTM Recommendet Version)
Debian 12 (UTM Recommendet)

![IMG_6303](https://github.com/user-attachments/assets/ae545638-a602-472e-b99b-6aca661364cb)
![IMG_6304](https://github.com/user-attachments/assets/1ee5e65b-a495-47b2-a17d-6d12bc0affb0)

**Comment by DragonSWDev** [comment]
For some reason AppStore UTM SE is unable to run things that are working fine on Mac version with same settings. Is there any particular reason why UTM SE compatibility is worse?

**Comment by Zack13358** [comment]
UTM SE (slow edition) is a lot slower because the appstore doesnt allow the use of JIT. I would recommend using a lighter weight guest OS for virtualization. https://github.com/utmapp/UTM?tab=readme-ov-file#utm-se

**Comment by Acceleratepowershellhell** [comment]
> UTM SE (slow edition) is a lot slower because the appstore doesnt allow the use of JIT. I would recommend using a lighter weight guest OS for virtualization. https://github.com/utmapp/UTM?tab=readme-ov-file#utm-se

Can you recommend a specific one?

**Comment by DragonSWDev** [comment]
@Zack13358 It's not only slower but compatibility is worse for some reason too. I tried running Windows NT 4 on UTM SE on iPad and installation crashed but with exactly same settings it worked fine on Mac version.

**Comment by Zack13358** [comment]
> @Zack13358 It's not only slower but compatibility is worse for some reason too. I tried running Windows NT 4 on UTM SE on iPad and installation crashed but with exactly same settings it worked fine on Mac version.

Yeah NT 4.0 is EXTREMELY picky on what it runs on, and win 9x doesn’t really like hypervisors / virtualization. I haven’t tested it yet but win 2k and / or win XP might be a little too heavy to run on UTM SE so really the best you’ll be able to do is win 3x -_-

**Comment by pixelkat5** [comment]
UTM SE is basically unusable due to Apple not allowing AppStore apps to use JIT. if you wanna use UTM SE you aren’t going to be able to really use anything past Windows XP. Even when I use XP for example it took me around 5+ hours and I still I haven’t even been able to actually get to a point where I can even look at the desktop. Anyways, 

TL:DR UTM SE is literally unusable and is only on the AppStore because well… why not. If you ask me it’s more of a joke than something to actually be used.
